src/authentication/contentAuth.ts

```diff
@@ -115,17 +115,18 @@ export class ContentAuth extends AlfrescoApiClient {
      *
      * @returns A promise that returns  if resolved and {error} if rejected.
      * */
-    validateTicket(): Promise<any> {
+    validateTicket(): Promise<string> {
         this.setTicket(this.config.ticketEcm);
 
-        let promise: any = new Promise((resolve, reject) => {
-            this.authApi.validateTicket().then((data: any) => {
+        let promise: any = new Promise<string>((resolve, reject) => {
+            this.authApi.validateTicket().then(
+                (data: any) => {
```

Comment:
Can we get rid of this `any`? According to api-explorer this endpoint should return 
<img width="523" alt="Zrzut ekranu 2023-08-14 o 07 32 14" src="https://github.com/Alfresco/alfresco-js-api/assets/113341662/bf690fe2-4c3d-4d26-961b-6ce984cdc5fa"> defined here: https://api-explorer.alfresco.com/api-explorer/?urls.primaryName=Authentication%20API#/authentication/validateTicket